samples/snippets/src/main/java/com/example/spanner/admin/generated/AddAndDropDatabaseRole.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.admin.generated;
+
+// [START spanner_add_and_drop_database_role]
+
+import com.google.cloud.spanner.admin.database.v1.DatabaseAdminClient;
+import com.google.common.collect.ImmutableList;
+import com.google.spanner.admin.database.v1.DatabaseName;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class AddAndDropDatabaseRole {
+
+  static void addAndDropDatabaseRole() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project";
+    String instanceId = "my-instance";
+    String databaseId = "my-database";
+    String parentRole = "parent_role";
+    String childRole = "child_role";
+    addAndDropDatabaseRole(projectId, instanceId, databaseId, parentRole, childRole);
+  }
+
+  static void addAndDropDatabaseRole(
+      String projectId, String instanceId, String databaseId, String parentRole, String childRole)
+      throws IOException {
+    final DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.create();
```

Comment:
These samples do not get the admin clients from a `Spanner` instance. That means that anyone who follows this sample would need to configure the emulator manually. I think that we would want to update these samples to use clients that will automatically use the emulator.

Also, this client is not used in a try-with-resources block, and it is also not closed in any other way. Good practice in a case like this would be to close the client when you are done.

Reply:
I am awaiting the BOM release before I can raise one PR which will update all the samples to use the new methods that we created.

Reply:
Makes sense, thanks!